server/views/pages/createAlertType/index.njk

```diff
@@ -10,6 +12,64 @@
                 <h1 class="govuk-heading-xl">
```

Comment:
Does this view or possibly the layout need a `govukErrorSummary`? I know this is primarily an admin portal and doesn't need to stick to GDS patterns necessarily

Reply:
Done.